spec/spec_helper.rb

```diff
@@ -6,19 +6,15 @@
 # ActiveRecord and ActiveJob require ActiveSupport, which affects a bunch of
 # core classes and may change some behavior that we rely on, so only bring it in
 # in some spec runs.
-if ENV['USE_RAILS'] == 'true'
```

Comment:
Just pointing out, the reason for `USE_RAILS` was to make sure that the gem still works if the application using it doesn't use ActiveSupport. In other words, to make sure that the tests will fail if any ActiveSupport-isms accidentally make it into the core gem.

The comment above explains this, but now it doesn't make any sense.

Reply:
Oh I totally missed that, I saw it in the circle ci configuration.

Should we add another dimension in the build matrix to test with and without this? On the other hand, this would make 36 build steps, what do you think?

Reply:
Instead of adding another dimension, we just test it once with `USE_RAILS` and once without. Probably easier to understand and reduces build time in general.

Reply:
Yeah, that's probably fine! I went on a run of getting the test time down as much as I could a while ago - that's why the CircleCI logic did a lot of test runs in parallel on only a few machines. But whatever makes sense for Github actions.

Reply:
I think around 2 minutes are pretty reasonable.

Also the main argument was to still have some kind of idea about all the builds, if you have a matrix of 36 builds, have fun 😂